src/models/GlobalStyleSheet.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+/* Wraps glamor's stylesheet and exports a singleton for global styles. */
+import { StyleSheet } from '../vendor/glamor/sheet'
+
+/* Don't specify a maxLength, since these rules are defined at initialization
+*  and should remain static after that */
+export default new StyleSheet({ speedy: false })
```

Comment:
Yeah leave this false for now. Inserting rules is faster but they're not slow atm—the problem with #400 is invalidation, not injection. And not being able to debug CSS in production is irritating af.